src/main/java/edu/kis/powp/command/FigureFactory.java

```diff
@@ -0,0 +1,44 @@
+package edu.kis.powp.command;
+
+import edu.kis.powp.jobs2d.Job2dDriver;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class FigureFactory {
+    public FigureFactory() {
+    }
+
+    public static ComplexCommand getRectangle(Job2dDriver driver, int x, int y, int width, int height) {
+        List<DriverCommand> commandList = new ArrayList<>();
+        commandList.add(new SetPositionCommand(x, y, driver));
+        commandList.add(new OperateToCommand(x + width, y, driver));
+        commandList.add(new OperateToCommand(x + width, y + height, driver));
+        commandList.add(new OperateToCommand(x, y + height, driver));
+        commandList.add(new OperateToCommand(x, y, driver));
+
+        return new ComplexCommand(commandList);
+    }
+
+    public static ComplexCommand getTriangle(Job2dDriver driver, int x, int y, int width, int height)
+    {
+        List<DriverCommand> commandList = new ArrayList<>();
+        commandList.add(new SetPositionCommand(x, y, driver));
+        commandList.add(new OperateToCommand(x + width/2, y - height, driver));
+        commandList.add(new OperateToCommand(x + width, y, driver));
+        commandList.add(new OperateToCommand(x, y, driver));
+
+        return new ComplexCommand(commandList);
+    }
+
+    public static ComplexCommand getHouse(Job2dDriver driver, int x, int y, int length)
+    {
+        List<DriverCommand> commandList = new ArrayList<>(getRectangle(driver, x, y, length, length).getCommandList());
+        commandList.addAll(FigureFactory.getTriangle(driver, x, y, length, length/2).getCommandList());
```

Comment:
why not simply commandList.add(FigureFactory.getTriangle(driver, x, y, length, length/2))
?